stubs/tensorflow/tensorflow/config/experimental.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import TypedDict
+
+from tensorflow.config import PhysicalDevice
+
+class _MemoryInfo(TypedDict):
+    current: int
+    peak: int
+
+def get_memory_info(device: str) -> _MemoryInfo: ...
+def reset_memory_stats(device: str) -> None: ...
+def get_memory_usage(device: PhysicalDevice) -> bytes: ...
```

Comment:
I think this returns an int. The documentation says `Total memory usage in bytes.` but means total is int like 5000 bytes.

Reply:
Fixed in 45ad7496c